src/main/java/core/basesyntax/CreateReport.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+public class CreateReport {
+
+    public String[] reportForFile(int allSupply, int allBuy, String res) {
+        return new String[] {WorkWithFile.SUPPLY + WorkWithFile.COMMA
+                + allSupply + System.lineSeparator(),
+                WorkWithFile.BUY + WorkWithFile.COMMA + allBuy + System.lineSeparator(),
+                res};
+    }
+}
```

Comment:
I think it would be better to move this logic to `WorkWithFile` class
after that we can make our constants `private`